odoo-shippable/scripts/build-image.sh

```diff
@@ -71,6 +71,13 @@ apt-get install ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
 # Install node dependencies
 npm install ${NPM_OPTS} ${NPM_DEPENDS}
 
+# Install virtualenv for each version of python
+for version in '2.7' '3.3' '3.4' '3.5'
```

Comment:
@moylop260 
I try to run `python3.6 -m pip install virtualenv` and show this error

![image](https://cloud.githubusercontent.com/assets/1387970/26108723/59027794-3a1b-11e7-8af9-5034eb9e4870.png)